manifests/config.pp

```diff
@@ -12,6 +12,26 @@
         changes => "set tftpd_flags '\"-s ${tftp::root}\"'",
       }
     }
+    if $facts['os']['family'] =~ /^(RedHat)$/ {
+      if versioncmp($facts['os']['release']['major'], '8') == 0 {
```

Comment:
I think it's actually safe to assume it's also true on newer versions (which may include Fedora):
```suggestion
      if versioncmp($facts['os']['release']['major'], '8') >= 0 {
```